labels.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import requests
+import json
+from webcompat import app
+
+API_URI = 'https://api.github.com/repos/'
+ISSUES_URI = app.config['ISSUES_REPO_URI'].rpartition('/issues')[0]
+USER_LABELS_URI = API_URI + ISSUES_URI + "/labels"
+WEBCOMPAT_LABELS_URI = API_URI + 'webcompat/webcompat-tests/labels'
+AUTH_HEADERS = {'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN']),
+                'User-Agent': 'webcompat/webcompat-bot'}
+
+
+def get_issue_labels(labels_uri):
+    '''Get the issue labels from the given repo.'''
+    r = requests.get(labels_uri)
+    return r.json()
```

Comment:
Here also you need to know if the request really happened. What is happening in the code if there's a network failure and the data are empty.